bench/thor/routes.cc

```diff
@@ -172,8 +172,8 @@ static void BM_UtrechtBidirectionalAstar(benchmark::State& state) {
 
   std::size_t route_size = 0;
 
+  thor::BidirectionalAStar astar;
   for (auto _ : state) {
```

Comment:
i assume its the iteration over this `state` object that lets the benchmark keep track of how long a run took?

Reply:
Yeah, that's the core benchmarking loop and how google bench controls the benchmark iterations.